task_imitation:
  object_inflation_size: 0.05 # How much to inflate each object's dimensions when doing collision checking, in meters
  match_dim_tolerance: 0.025 # When auto-initializing object poses, objects whose box dimensions are within this tolerance of the model's dimensions are considered a possible match
  cart_path_eef_threshold: 0.02 # The maximum step size between end effector positions when following a trajectory
  cart_path_jump_threshold: 4 # The jump_threshold for planning paths when following a trajectory (measured in joint space distance)
  slice_pause_duration: 1 # Amount of time, in seconds, to wait in between executing slices
